documentation/book/con-kafka-connect-authentication.adoc

```diff
@@ -2,12 +2,16 @@
 //
 // assembly-kafka-connect-tls.adoc
 
-[id='ref-kafka-connect-authentication-{context}']
+[id='con-kafka-connect-authentication{context}']
 = Authentication support in Kafka Connect
 
 Authentication can be configured in the `authentication` property in `KafkaConnect.spec` and `KafkaConnectS2I.spec`.
 The `authentication` property specifies the type of the authentication mechanisms which should be used and additional configuration details depending on the mechanism.
-Currently, the only supported authentication type is TLS client authentication.
+Currently, the supported authentication types are:
```

Comment:
Currently, the supported authentication .... > The supported authentication ....